src/plugins/resolve-media-queries-plugin.js

```diff
@@ -73,7 +74,7 @@ function _topLevelRulesToCSS({
     );
 
     // CSS classes cannot start with a number
-    const mediaQueryClassName = 'rmq-' + hash(query + ruleCSS);
+    const mediaQueryClassName = `rmq-${styleId()}-${hash(query + ruleCSS)}`;
```

Comment:
`styleId()` will execute every render. Since `addCSS` no longer dedupes due to the incrementing number, you'll end up with many duplicate styles in the CSS, right?